index.html

```diff
@@ -0,0 +1,256 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>MOBILE VERSION</title>
+    <script src="https://kit.fontawesome.com/de9dd03ce2.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <!---POPPINS-->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap" rel="stylesheet">
+    <!--CRETE ROUND-->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+    <!--INTER-->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <section class="wrap-section">
+        <header>
+            <nav>
+                <a href="#" class="my-logo">Aneal Laryea</a>
+                <i class="fa-solid fa-bars"></i>
+            </nav> 
+        </header>
+    
+        <section class="headline">
+            <div class="primary-text">
+                <h1>Hey There.
+                    I’m Aneal</h1>
+            </div>
+            <div class="supporting-text">
+                <h5>I’am a Software Developer</h5>
+            </div>
+            <div class="description">
+                <h5>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.</h5>
+            </div>
+            <div class="social-media">
+                <ul class="socials">
+                    <li class="ss"><i class="fa-brands fa-github"></i></li>
+                    <li class="ss"><i class="fa-brands fa-linkedin"></i></li>
+                    <li class="ss"><i class="fa-brands fa-angellist"></i></li>
+                    <li class="ss"><i class="fa-brands fa-twitter"></i></li>
+                    <li class="ss"><i class="fa-solid fa-m"></i></li>
+                </ul>
+            </div>
+        </section>
+
+         <!--work section-->
+        <section class="work-section">
+            <!--frame 44-->
+            <div class="title">
+               <h1 class="title-text">My Recent Works</h1>
+           </div>
```

Comment:
- [ ] Kindly make this a `h2` tag. It is always a good convention to have a single `h1` tag on your entire webpage. It's good for hierarchy and accessibility reasons.
- [ ] Kindly change all your `h1` tags to `h2/h3` tags except for the one in the headline section. That's the only one meant to be a `h1` tag. This will become very useful in the near future in this project 👍